### PR TITLE
INTG-573 Added fixes from QA

### PR DESCRIPTION
**Pull request by jwsandeman**

### CODE REVIEW

**Review by rossi-mx51 — CHANGES_REQUESTED** [review]

File: src/components/OrderConfirmation/PreAuthOrderConfirmation.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ const PreAuthOrderConfirmationComponent: React.FC<IProps> = ({ setShowTransactio
     }
   };
 
-  function isAccountVerify() {
+  function isDisabled() {
```
[the remark this answers is not in the document]
Comment (jwsandeman) [reply src/components/OrderConfirmation/PreAuthOrderConfirmation.tsx:80]: Good suggestion, done.

File: src/components/PaymentSummary/PaymentSummary.tsx [hunk 2]
```diff
@@ -67,25 +71,41 @@ function PaymentSummary(): React.ReactElement {
...
+    PCOMP: 'Preauth Complete',
+    'PRE-AUTH EXT': 'Preauth Extend',
+    'PRE-AUTH CANCEL': 'Preauth Cancel',
+    TOPUP: 'Preauth Topup',
+    'A/C VERIFIED': 'Account Verified',
+    CANCEL: 'Preauth Reduce',
+    'PRE-AUTH': 'Preauth Open',
+  };
+
+  const transactionStatus = () => {
```
[the remark this answers is not in the document]
Comment (jwsandeman) [reply src/components/PaymentSummary/PaymentSummary.tsx:84]: good suggestion, done.

File: src/hooks/usePreAuthActions.tsx [hunk 3]
```diff
@@ -31,6 +31,8 @@ export const usePreAuthActions = (currentTerminal: ITerminalProps): IPreAuthActi
   const handlePreAuthActions = (keyPadAmount?: number, surchargeAmount?: number, preAuthRefId?: string) => {
     if (keyPadAmount) {
```
[the remark this answers is not in the document]
Comment (jwsandeman) [reply src/hooks/usePreAuthActions.tsx]: Yeah good idea, ive broken this down into 3 separate functions now all with a single purpose.

**Review by rossi-mx51 — APPROVED** [review]
Great work 👏

**Pull request merged**